frontend/src/scenes/session-recordings/player/list/PlayerListRow.tsx

```diff
@@ -115,8 +118,11 @@ function PlayerListRowRaw<T extends Record<string, any>>({
                             <IconWindow value="1" className="text-muted shrink-0" />
                         </div>
                     </div>
-                    <div className="grow overflow-hidden">{contentDetermined}</div>
-                    <div className="flex shrink-0 flex-row gap-3 items-center text-muted">
+                    <div className={clsx('grow h-full', !isExpanded && 'overflow-hidden')}>{contentDetermined}</div>
+                    <div
+                        className="flex shrink-0 flex-row gap-3 items-center text-muted"
+                        style={{ lineHeight: '1.5rem' }}
```

Comment:
Interested by these lineHeights everywhere. Fine for now but might be something to come back to later and question / move to tailwind-esque helpers

Reply:
We have `leading-X` tailwind classnames for line heights but these don't override the ones set by `text-X`. These were set so that the content would be consistently spaced in unexpanded and expanded states — but open to different tailwind friendly implementations.

Reply:
Ah yeah that makes sense. I just committed a change that takes care of this. We just needed the .leading utility generators to come after the text ones which is what Tailwind does as well